lesson15/js/main.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+const openModalBtn = document.getElementById('js-openModalBtn');
+openModalBtn.addEventListener(
+	'click',
+	() => {
+		const getModal = document.getElementById('js-modalContainer');
+		getModal.classList.add('is-visible');
+		const modalBg = document.createElement('div');
+		modalBg.id = 'js-bgFixed';
+		modalBg.classList.add('el_modalBg');
+		document.body.appendChild(modalBg);
+		document.body.style.position = 'fixed';
+		closeButton();
+	},
+	false
+);
+
+function closeModal() {
+	const getModal = document.getElementById('js-modalContainer');
+	getModal.classList.remove('is-visible');
+	const bgFixed = document.getElementById('js-bgFixed');
+	document.body.removeAttribute('style');
+	bgFixed.remove();
+}
+
+function closeButton() {
+	const closeModlBtn = document.getElementById('js-closeModal');
+	closeModlBtn.addEventListener(
+		'click',
+		() => {
+			closeModal();
+		},
+		false
+	);
+}
+
+const form = document.getElementById('js-loadmenuBtn');
+form.addEventListener(
+	'click',
+	(event) => {
+		event.preventDefault();
+		const inputNumber = document.getElementById('number');
+		const inputText = document.getElementById('words');
+		if (!inputNumber.value === false && !inputText.value === false) {
+			outputForm();
+			closeModal();
+			callMenuContents();
+		} else {
+			alert('未入力です。');
+		}
```

Comment:
IMO
I think it would be easier to declare when the value is truthy.
```suggestion
		if (inputNumber.value && inputText.value) {
			outputForm();
			closeModal();
			callMenuContents();
		} else {
			alert('未入力です。');
		}
```

Reply:
Thank you fore review!
I was able to write it more simply by the method you taught me.
The input text has been made negative so that it can handle the blanks pointed out by Sae.

[1f06d7a](https://github.com/nyattsu72/hanson-frontend-cource/pull/20/commits/1f06d7a893a112df037215aa8842fa75acb9b1f5)